app/components/Home.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react';
+const { EditorKit, EditorKitDelegate } = require('sn-editor-kit');
+const MarkdownIt = require('markdown-it');
+
+const EditMode = 0;
+const SplitMode = 1;
+const PreviewMode = 2;
+
+const modes = [
+  { mode: EditMode, label: "Edit", css: "edit" },
+  { mode: SplitMode, label: "Split", css: "split" },
+  { mode: PreviewMode, label: "Preview", css: "preview" },
+];
+
+const markdownitOptions = {
+  // automatically render raw links as anchors.
+  linkify: true
+};
+
+const MarkdownParser = MarkdownIt(markdownitOptions)
+  .use(require('markdown-it-footnote'))
+  .use(require('markdown-it-task-lists'))
+  .use(require('markdown-it-highlightjs'));
+
+type AppProps = {
+  text?: string,
+  mode: any;
+  platform?: any,
+}
+
+const initialState = {
+  mode: modes[0],
+  text: "",
+}
+const debugMode = false;
+
+if (debugMode) {
+  console.log("home loaded 2");
```

Comment:
This should either be deleted or edited to be a more helpful console message. Definitely shouldn't include arbitrary numbers ;)

Reply:
I'll remove them